recipes/coincurve/meta.yaml

```diff
@@ -0,0 +1,101 @@
+{% set name = "coincurve" %}
```

Comment:
This is a duplicate recipe. Coincurve has it's own PR. Please combine all of your related PRs into one or wait until the coincurve PR is merged.

Reply:
I need to re-think how I organize the recipes I submitted. I do want to test them through the flow (in particular since windows is so foreign to me).

Is the best way to move this recipe (and the other similar ones) back to draft?

How to remove the `review-requested` flag to avoid reviewers going through the recipes that are moved back to draft?

From your experience, and given that all the recipes are passing checks, what would be the more productive strategy?
   - Focus on individual packages and having to wait through all the individual PRs to be merged
   - Focus on the final packages that I need (with the 5+ individual packages bundled)?
   
Thank you

Reply:
I think I have to remove the labels for you because you aren't a member of the conda-forge org yet. Yes, reverting them to a draft is a good idea. You can also debug your recipes on your local machine (if you have a Windows machine) because you can run conda-build from the command line. Just use the conda-forge channel for all of the packages and the `--use-local` flag.

You should consider the dependency tree of the ecosystem that you are trying to build and break it into PRs by theme/dependency For example, all the binding packages in one PR, coinbase and the libsec256k1 in another PR etc.

Reply:
I have used some of the `build-locally.py`, but it is not foolproof (for example for locally installed packages) and I do not have a mac or windows (which is really a tremendous pain for me, likely because I know very little about it)

If you don't mind, can we start with `libsecp256k1`? (https://github.com/conda-forge/staged-recipes/pull/23882). This is the one I worked on the most

The `coincurve_dynamic` is then the second one that I think simplifies and implements a few of the feedback you gave me so far

I don't think it is worth our time to work on `coincurve` since `coincurve_dynamic` is much simpler, I would rather close it